frontend/app/app.css

```
@@ -47,6 +47,17 @@ body {
   }
 }
 
+@keyframes slideInRight {
```

Stylelint (17.10.0)

[error] Expected keyframe name "slideInRight" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)